ncm-opennebula/src/test/perl/OpennebulaMock.pm

```diff
@@ -35,6 +36,9 @@ foreach my $getnam ("getpwnam", "getgrnam") {
     $nco->mock( $getnam, sub { return 1; } );
 }
 
+$nco->mock('detect_opennebula_version', sub { return version->new("5.0.0"); } );
```

Comment:
no need to mock like this, fake the filecontents with `set_file_content("/.../VERSION", "5.0.0")`